<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="dial_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="last_message_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard</set>
   </property>
  </widget>
  <widget class="QLabel" name="photo_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dial_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="dial_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="name_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dial_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="name_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dial_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="name_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dial_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="name_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dial_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="name_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dial_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="name_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>520</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dial_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="name_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>570</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dial_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">dialogue_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="name_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>590</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_message_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>620</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Мои сообщения:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>🔎 Введите имя или id собеседника</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="message_input">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>810</y>
     <width>801</width>
     <height>51</height>
    </rect>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_file">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>810</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>📎</string>
   </property>
  </widget>
  <widget class="QLabel" name="current_user">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Name Surname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="current_user_photo">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>40</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_block">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Block user</string>
   </property>
  </widget>
  <widget class="QLabel" name="your_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>870</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Your id: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="messages">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>801</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>dial_1</zorder>
  <zorder>last_message_1</zorder>
  <zorder>name_1</zorder>
  <zorder>photo_1</zorder>
  <zorder>dial_2</zorder>
  <zorder>dial_3</zorder>
  <zorder>name_3</zorder>
  <zorder>last_message_3</zorder>
  <zorder>photo_3</zorder>
  <zorder>dial_4</zorder>
  <zorder>name_4</zorder>
  <zorder>last_message_4</zorder>
  <zorder>photo_4</zorder>
  <zorder>dial_5</zorder>
  <zorder>name_5</zorder>
  <zorder>last_message_5</zorder>
  <zorder>photo_5</zorder>
  <zorder>dial_6</zorder>
  <zorder>name_6</zorder>
  <zorder>last_message_6</zorder>
  <zorder>photo_6</zorder>
  <zorder>dial_7</zorder>
  <zorder>name_7</zorder>
  <zorder>last_message_7</zorder>
  <zorder>photo_7</zorder>
  <zorder>dial_8</zorder>
  <zorder>name_8</zorder>
  <zorder>last_message_8</zorder>
  <zorder>photo_8</zorder>
  <zorder>dial_9</zorder>
  <zorder>name_9</zorder>
  <zorder>last_message_9</zorder>
  <zorder>photo_9</zorder>
  <zorder>dial_10</zorder>
  <zorder>name_10</zorder>
  <zorder>last_message_10</zorder>
  <zorder>photo_10</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>message_input</zorder>
  <zorder>button_file</zorder>
  <zorder>current_user</zorder>
  <zorder>current_user_photo</zorder>
  <zorder>button_block</zorder>
  <zorder>your_id</zorder>
  <zorder>last_message_2</zorder>
  <zorder>name_2</zorder>
  <zorder>photo_2</zorder>
  <zorder>messages</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="dialogue_buttons"/>
 </buttongroups>
</ui>
